bin/bin.go

```diff
@@ -0,0 +1,250 @@
+package bin
```

Comment:
"bin" is a bit of a reserved word in cs. I worry that users looking at a bin package may expect it to handle binary information. If everything in here handles fastas, can this go in that package? 
I worry someone looking for a tool to bin fastas might want a more descriptive name like faBin and it will be easier to find that way

Reply:
that's a great point, I can go with something like faBin for sure for the command and change the package name. I was originally going to put it in the fasta package, but when Craig and I spoke we waffled a bit on that point, happy to put it there if that's the preferred home for it